instr.ml

```diff
@@ -398,12 +404,13 @@ class map = object (m)
       Assert (m#expression e)
     | Stop e ->
       Stop (m#expression e)
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       Osr {
         label;
```

Comment:
I am favorable to anti-over-engineering moves in general, but to be fair this is a case where the visitor has a *very* regular structure that always visits all arguments (I realize now that `op` is not visited, but that is a change you introduced), and I think moving away from this regular structure is wrong (before it was very simple to know what to do, and now people have to think about what to do in each case).

Besides, in that case visiting the label is precisely required if you want to freshen all the variables and labels of a piece of code. As for the `variable` case, we should have a generic `label` method that all three of `branch_label`, `goto_label` and `osr_label` defer to. I can send a PR after this one is merged.